FreedomStudio/E31FPGA/dhrystone/sifive-coreplexip-e31-arty.cfg

```diff
@@ -0,0 +1,31 @@
+# JTAG adapter setup
```

Comment:
The new version of OpenOCD no longer uses this file, functionality was split between interface and board.

Reply:
Sure, we will update to the config files included in the openocd builds in the near future.